src/metabase/models/secret.clj

```diff
@@ -126,7 +130,9 @@
         sub-prop #(keyword (str connection-property-name "-" (name %)))]
     (zipmap sub-prop-types (map sub-prop sub-prop-types))))
 
-(def ^:private uploaded-base-64-pattern #"^data:application/([^;]*);base64,")
+(def uploaded-base-64-pattern
+  "Regex for parsing base64 encoded file uploads."
+  #"^data:application/([^;]*);base64,")
```

Comment:
Might be good to put the $ on the end of this regex, just in case someone uses this with re-find.

```suggestion
  #"^data:application/([^;]*);base64,$")
```

Reply:
I am using it with re-find in this PR. The upload data looks something like `data:application/x-pkcs12;base64,MIIMIQIBAzCCC+cGCSqGSIb3DQEHAaCCC9gEggv...` (a long base64 string) so this is just for parsing out the prefix of it; it's intentionally not matching on the entire string. I'll change the name of this var to make that clearer.